x-element.js

```diff
@@ -68,7 +87,9 @@ export default class XElement extends HTMLElement {
    * E.g., setup focus delegation or return host instead of host.shadowRoot.
    */
   static createRenderRoot(host) {
-    return host.attachShadow({ mode: 'open' });
+    const shadowRoot = host.attachShadow(this.shadowRootInit);
+    shadowRoot.adoptedStyleSheets = this.styleSheets;
```

Comment:
I recommend moving this out of `createRenderRoot` and instead moving it to the constructor — it would throw an error if there is no `shadowRoot` on the host